Autopilot/DataStructures/Inc/vectorClass.hpp

```diff
@@ -0,0 +1,48 @@
+/*
+    Author: Nixon Chan
+*/
+
+#ifndef VECTOR_CLASS_HPP
+#define VECTOR_CLASS_HPP
+
+class Vector3D
+{
+    private:
+        double x,y,z;
```

Comment:
rather than having 3 getter and 3 setter methods, why not just make these variables public? then people can access vector.x really easily and cleanly.

Reply:
Fixed!